panel/pane/markup.py

```diff
@@ -185,6 +184,8 @@ class DataFrame(HTML):
 
     _stylesheets = ['css/dataframe.css']
 
+    _default_styles = {"overflow": "auto", "padding-right": "1px"}
```

Comment:
You should be able to put this in `css/dataframe.css` as:

```css
:host {
    overflow: auto;
    padding-right: 1px;
}
```

Reply:
Thanks. I've applied it.

I also took the liberty to align the hover background color with the one in Jupyter lab. Its prettier. And then its consistent.